pytest_marker_bugzilla.py

```diff
@@ -144,12 +144,9 @@ def pytest_runtest_setup(self, item):
             return
 
         bugs = item.funcargs["bugs"]
-        will_skip = True
         skippers = []
         for bz in bugs.bugs_gen:
-            if bz.status not in ["NEW", "ASSIGNED", "ON_DEV"]:
-                will_skip = False
-            else:
+            if bz.status in ["NEW", "ASSIGNED", "ON_DEV"]:
```

Comment:
At some point, this should be configurable because not every BZ set up is the same so these states may not be applicable to all BZ users.